src/platform/configuration/config/template/user.template.interface.ts

```diff
@@ -10,6 +10,5 @@ export interface IReferenceTemplate {
 }
 export interface IUserTemplate {
   name: string;
-  tagsets?: ITagsetTemplate[];
```

Comment:
How is this change related to the PR?

Reply:
Ditto re comment on org template: the tagsets are now created by the entity, not by the client